### ISSUE TITLE
[Request] Add example code to the wiki

### ISSUE THREAD

**Issue by Silly511** [Chisel-Team/CTMLib#4]

**Comment by tterrag1098** [comment]
There are some base implementations for most things already.

* [Block](https://github.com/Chisel-Team/CTMLib/blob/dev/src/main/java/team/chisel/ctmlib/CTMBlock.java)
* [Submap Manager](https://github.com/Chisel-Team/CTMLib/blob/dev/src/main/java/team/chisel/ctmlib/SubmapManagerCTM.java)

**Comment by Silly511** [comment]
Yeah but nothing on what code to use. Somethings are obvious like extending CTMBlock, but other things like having to use CTMRenderer are hard to figure out. I got stumped for 3 hours before I figured out I needed to register CTMRenderer.
